rust/src/client/mod.rs

```diff
@@ -108,8 +133,9 @@ impl Client {
     /// `id`: an ID to assign to the [`Client`].
     /// `handle`: handle for communicating with the (local) service.
```

Comment:
i'd suggest to add line breaks or an enumeration, because markdown writes this all in one line. similarly for some other methods below.